user_docs/en/userGuide.md

```diff
@@ -2169,7 +2171,7 @@ If "Read by paragraph" is checked, the selected start marker will be displayed t
 This can be especially helpful in applications used to read large pieces of text, like structured documents or books.
 In such documents, knowing where paragraphs start may be useful to understand the structure of the content, or to set bookmarks or annotations based on paragraph position.
 
-The options include using double space for softer paragraph breaks, or alternatively the paragraph symbol, Pilcrow (¶), for more obvious paragraph breaks.
+The options include using double-spaces for softer paragraph breaks, and the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.
```

Comment:
I actually think this sentence could do with a bit more work. I'm not fond of repeating "paragraph breaks" multiple times in one sentence. And I think we can simplify "double-spaces" to "two spaces". I've also changed "soft" to "subtle".

```suggestion
The options include using two spaces as a subtle paragraph break, and the paragraph symbol, pilcrow (¶), as a more obvious one.
```

What do you think, @XLTechie, @CyrilleB79, @Qchristensen?

Reply:
Do we need to explain why you might use each?  What about:
```suggestion
The options include using double-spaces or the paragraph symbol, pilcrow (¶).
```

Reply:
I prefer Sasha's proposal, i.e. do not drop the use case for each type of paragraph marker. Indeed, describing the use cases in the user guide can help the user make their choice.

Describing the use cases in the user guide is not necessary when the use case is obvious from the option. But in the present description, I feel that the use case description is a plus.

Reply:
Also cc @nvdaes, author of this change.

Reply:
I also prefer Sascha's proposal. I for one would not have known why I should choose one over the other, if it hadn't been written with this moderate explanation.